### DISCUSSION TITLE
Suddenly not able to use the free tier anymore

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#6023] [Help]
![Screenshot 2022-05-23 at 12 24 30 PM](https://user-images.githubusercontent.com/1195573/169743072-eafdb8dd-8002-43c8-b530-743ee7335ae8.jpg)

I have been exploring Budibase for a few days but suddenly I cant use Budibase anymore. I only have:
- 2 apps
- 1 automation
- hardly any data, like <5 rows of records in each app. 

Any idea what kind of limit I am hitting?

**Comment by melohagan** [comment]
Hi @magickaito 

Would you be able to provide your tenant id (Budibase domain) please?

Then we can investigate further and get back to you.

**Reply by magickaito** [reply]
Sure my domain is tplus.budibase.app
